pkg/deploy/api/test/ok.go

```diff
@@ -160,6 +160,36 @@ func OkPodTemplate() *kapi.PodTemplateSpec {
 	}
 }
 
+func OkPodTemplateMissingImageWithOkImageChangeTrigger() *kapi.PodTemplateSpec {
```

Comment:
Can't you just call `OkPodTemplate()` and remove the image name instead of duplicating all the podtemplate code?